retroarch.c

```diff
@@ -21578,12 +21578,8 @@ static void video_driver_frame(const void *data, unsigned width,
    if (!video_driver_active)
       return;
 
-   if (data)
-      frame_cache_data = data;
-   frame_cache_width   = width;
-   frame_cache_height  = height;
-   frame_cache_pitch   = pitch;
-   
+   video_driver_cached_frame_set(data, width, height, pitch);
```

Comment:
Bunch of whitespace changes, but this is the change.

Reply:
Yeah. Sorry about the trailing whitespace removals, vscode did that for me.

Reply:
Merged master to fix whitespace changes.